tcp_queue_length/README.md

```diff
@@ -0,0 +1,61 @@
+# Agent Check: TCP Queue Length
+
+## Overview
+
+This check monitors the size of the Linux TCP receive and send queues.
+
+## Setup
+
+### Installation
+
+`tcp_queue_length` is a core Agent 6/7 check that relies on an eBPF part implemented in `system-probe`.
+
+The eBPF program used by `system-probe` is compiled at runtime and requires you to have access to the proper kernel headers.
+
+On Debian-like distributions, install the kernel headers like this:
+```sh
+apt install -y linux-headers-$(uname -r)
+```
+
+On RHEL-like distributions, install the kernel headers like this:
+```sh
+yum install -y kernel-headers-$(uname -r)
+```
+
+### Configuration
+
+Enabling it requires to enable it both for `system-probe` and the core agent.
+
+Inside the `system-probe.yaml` configuration file, the following parameter must be set:
+```yaml
+system_probe_config:
+  enable_tcp_queue_length: true
+```
+
+For the “core” agent, a `tcp_queue_length.d/conf.yaml` file is needed in the `conf.d` directory [with the following content][1].
+
+The `only_count_nb_contexts` parameter controls whether:
+* the check should collect all the data—that is, the size of the TCP queues for every single connection (`false`)
+* the check should collect only the number of timeseries it would normally collect (`true`). This is the default.
+
+### Configuration with Helm
+
+With the [Datadog Helm chart][2], the `datadog.systemProbe.enableTCPQueueLength` parameter is the only thing to set in `values.yaml` to enable the check.
```

Comment:
should we mention that both the system probe and process agent should be enabled too?

Reply:
Indeed!
I added the parameter to activate `system-probe`.
The `process-agent`, on the other hand isn’t needed.